packages/cloudflare/README.md

```diff
@@ -222,21 +240,108 @@ import cloudflare from '@astrojs/cloudflare';
 export default defineConfig({
   output: 'server',
   adapter: cloudflare({
-+   runtime: { mode: 'local' },
++   runtime: {
++     mode: 'local',
++     type: 'pages',
++     bindings: {
+        // example of a KV binding
++       "URL": {
++         type: "var",
++         id: "https://example.com",
++       },
+        // example of a KV binding
++       "KV": {
++         type: "kv",
++         id: "KV",
++       },
+        // example of a D1 binding
++       "D1": {
++         type: "d1",
++         database_name: "D1"
++         preview_database_id: "D1",
++         database_id: "D1",
++       },
+        // example of a R2 binding
++       "R2": {
++         type: "r2",
++         bucket_name: "R2",
++       },
+        // example of a Durable Object binding
++       "DO": {
++         type: "durable-object",
++         className: "DO",
++       },
++     },
++   },
   }),
 });
 ```
 
-## Cloudflare runtime
+If you also need to define `secrets` in addition to enviroment variables, you need to add a `.dev.vars` file to the root of the Astro project:
 
-Gives you access to [environment variables](https://developers.cloudflare.com/pages/platform/functions/bindings/#environment-variables), and [Cloudflare bindings](https://developers.cloudflare.com/pages/platform/functions/bindings).
+```
+# .dev.vars
+DB_PASSWORD=myPassword
+```
 
-Currently supported bindings:
+If you want to use `wrangler` for cli commands, e.g. D1 migrations, you would also need to add a `wrangler.toml` to the root of the Astro project with the correct content. Consult [Cloudflare's documentation](https://developers.cloudflare.com/) for further details.
 
-- [Cloudflare D1](https://developers.cloudflare.com/d1/)
-- [Cloudflare R2](https://developers.cloudflare.com/r2/)
-- [Cloudflare Workers KV](https://developers.cloudflare.com/kv/)
-- [Cloudflare Durable Objects](https://developers.cloudflare.com/durable-objects/)
+```toml
+# wrangler.toml
+name = "example"
+compatibility_date = "2023-06-14"
+
+# example for D1 Binding
+[[d1_databases]]
+binding = "D1"
+database_name = "D1"
+database_id = "D1"
+preview_database_id = "D1"
+```
+
+#### Cloudflare Workers
+
+Compared to [Cloudfare Pages](#cloudflare-pages), for Cloudflare Workers you need to configure the bindings using a `wrangler.toml` config file in the root directory of your Astro project. This file is also needed for production deployments.
```

Comment:
```suggestion
Unlike for [Cloudfare Pages](#cloudflare-pages), for Cloudflare Workers you need to configure the bindings using a `wrangler.toml` config file in the root directory of your Astro project. This file is also needed for production deployments.
```

I don't think "compared to" is what you want here, because this didn't immediately give me a comparison. I suggested something above that I think might be close to what you mean, but the issue is that use of "compared to".

Note that it's not even really necessary to relate it to Cloudflare pages at all, since you can imagine some people will turn up on this page, look at the sidebar and think, "Oh, I want Cloudflare Workers!" and just click to get right here, never having read anything about Cloudflare Pages!  :smile:  You could totally just start this sentence at "For Cloudflare Workers, you need to..." and, that would be fine!